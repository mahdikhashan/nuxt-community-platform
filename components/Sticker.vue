<template>
    <div
    class="
      flex
      bg-gray-200
      items-center
      justify-between
      p-4
      rounded-md
      border-gray-700
      border-[1px]
    "
  >
    <div class=""><slot></slot></div>
  </div>
</template>

<script>
export default {
    name: "Sticker",
}
</script>