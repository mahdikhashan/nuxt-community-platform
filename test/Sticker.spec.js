import { mount } from '@vue/test-utils'
import Sticker from '@/components/Sticker.vue'

describe('Sticker', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Sticker)
    expect(wrapper.vm).toBeTruthy()
  })
})
