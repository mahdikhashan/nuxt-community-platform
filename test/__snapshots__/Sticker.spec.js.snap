// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sticker is a Vue instance 1`] = `
"<div class=\\"
    flex
    bg-gray-200
    items-center
    justify-between
    p-4
    rounded-md
    border-gray-700
    border-[1px]
  \\">
  <div></div>
</div>"
`;
